docs/fx-core/bot-help.md

```diff
@@ -8,6 +8,25 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
 
 Please refer to this [link](https://aka.ms/rps-not-found) to register your subscription to use namespace 'Microsoft.BotService'.
 
+## BT.BotRegistrationNotFoundError
+
+### Error Message
+
+Bot registration was not found. Click 'Get Help' button to learn more about how to check bot registrations.
+
+### Mitigation
+
+The first step is to make sure the remote bot registration is not existing.
+Suppose your Teams app's name is `someTeamsApp`, then the target bot registration name you are supposed to find is something like `someTeamsApp-local-debug`. So follow the steps below:
+1. Login to [Bot Framework Dev Portal](https://dev.botframework.com/) with your M365 account which is the same account you used in Teams Toolkit.
```

Comment:
In our document, we need to apply and align the terminology, making "M365" -> "Microsoft 365"